src/browser/BrowserService.h

```diff
@@ -25,7 +25,7 @@
 #include "gui/DatabaseTabWidget.h"
 #include "core/Entry.h"
 
-enum { max_length = 16*1024 };
+enum { max_length = 1024*1024 };
```

Comment:
You should define this in NativeMessagingBase.h since it is shared between the proxy and browser code. Also, define as a constant: ```static const int NATIVE_MSG_MAX_LENGTH = 1024*1024;``` with all caps being the standard convention